### PR TITLE
Clarify js library expectations

### PR DESCRIPTION
**Pull request by orangejenny**
@emord / @calellowitz

### CODE REVIEW

File: libraries.md [hunk 1]
```diff
@@ -1,6 +1,9 @@
 # Third-Party Libraries
 
-- jQuery: Available throughout HQ. We use [jQuery 3](https://github.com/dimagi/commcare-hq/blob/master/bower.json) everywhere except in Web Apps, which depends on [marionette](https://marionettejs.com/) v2, which is only compatible with jQuery 2.
-- Knockout: available throughout HQ. Should be on Knockout 3.0, although there are a few places that haven't yet been upgraded from 2.x.
-- Backbone: Used in CloudCare. Prefer Knockout for new code.
-- Angular: Used in a few places (dashboards, exports, app summaries). Prefer Knockout for new code.
+jQuery is available throughout HQ. We use [jQuery 3](https://github.com/dimagi/commcare-hq/blob/master/bower.json) everywhere except in Web Apps, which depends on [marionette](https://marionettejs.com/) v2, which is only compatible with jQuery 2.
```
Comment (emord) [inline libraries.md]: Should jquery 3's link really be to bower.json? all the other links go to external sites about the library. bower.json seems more like a general one

Comment (orangejenny) [reply libraries.md]: Yes, updated. I think this used to link to the specific line but that's not maintainable.

**Comment by orangejenny** [comment]
added underscore

File: libraries.md (hunk 1 above)
Comment (emord) [reply libraries.md]: Maybe the heading could be a link to the bower.json? or just have a general link to it as an exhaustive list?

**Pull request merged**